UnitTests/UnitTests.Shared/Mvvm/Test_ObservableValidator.cs

```diff
@@ -0,0 +1,173 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using Microsoft.Toolkit.Mvvm.ComponentModel;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitTests.Mvvm
+{
+    [TestClass]
+    public class Test_ObservableValidator
+    {
+        [TestCategory("Mvvm")]
+        [TestMethod]
+        public void Test_ObservableValidator_HasErrors()
+        {
+            var model = new Person();
+
+            Assert.IsFalse(model.HasErrors);
```

Comment:
Should this be true since Person.Name is required?

Reply:
Waiting for your reply on the comment above (https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3429#discussion_r487103283) to decide how to proceed with this. Right now this is expected to be false as no validation has been performed yet, so there are no logged errors at this point.